docs/templates/resource.rst.jinja

```diff
@@ -7,12 +7,44 @@
 Access methods
 ^^^^^^^^^^^^^^^
 
-{% if resource.create_database_schema -%}
-* `Browse or query this table in Datasette. <https://data.catalyst.coop/pudl/{{ resource.name }}>`__
-{% else -%}
-* This table is not published to Datasette.
-{%- endif %}
-* `Download this table as a Parquet file. <https://s3.us-west-2.amazonaws.com/pudl.catalyst.coop/nightly/{{ resource.name }}.parquet>`__
+* `Browse and query this table online <https: //viewer.catalyst.coop/search?&q=name:{{ resource.name }}>`__
+* `Download this table as a Parquet file <https: //s3.us-west-2.amazonaws.com/pudl.catalyst.coop/nightly/{{ resource.name }}.parquet>`__
+
+.. tabs::
+
+   .. tab:: pandas
+
+      .. code-block:: python
+
+         import pandas as pd
+         df = pd.read_parquet(
+             "s3://pudl.catalyst.coop/nightly/{{ resource.name }}.parquet",
+             dtype_backend="pyarrow",
+         )
```

Comment:
Manually inserting (ungrouped) code-blocks rather than using the (automatically grouped) `code-tabs` shortcut seems to have addressed the crazy scrolling.